mattermost.tera

```diff
@@ -0,0 +1,34 @@
+---
+whiskers:
+  version: "2.2.0"
+  matrix:
+    - flavor
+  filename: "themes/catppuccin-{{ flavor.identifier }}.json"
+---
+{
+  "sidebarBg": "#{{ mantle.hex }}",
+  "sidebarText": "#{{ text.hex }}",
+  "sidebarUnreadText": "#{{ green.hex }}",
+  "sidebarTextHoverBg": "#{{ overlay0.hex }}",
+  "sidebarTextActiveBorder": "#{{ overlay0.hex }}",
+  "sidebarTextActiveColor": "#{{ base.hex }}",
+  "sidebarHeaderBg": "#{{ crust.hex }}",
+  "sidebarHeaderTextColor": "#{{ text.hex }}",
+  "onlineIndicator": "#{{ green.hex }}",
+  "mentionBg": "#{{ pink.hex }}",
+  "awayIndicator": "#{{ peach.hex }}",
+  "dndIndicator": "#{{ red.hex }}",
+  "mentionBj": "#{{ crust.hex }}",
+  "mentionColor": "#{{ crust.hex }}",
+  "centerChannelBg": "#{{ base.hex }}",
+  "centerChannelColor": "#{{ subtext0.hex }}",
+  "newMessageSeparator": "#{{ red.hex }}",
+  "linkColor": "#{{ rosewater.hex }}",
```

Comment:
Let's change this to a refactor/fix and change this from rosewater to blue while we're here, rosewater links were a mistake :cry: 
```suggestion
  "linkColor": "#{{ blue.hex }}",
```
Or if you'd like to PR this in separately then good with that too.